src/main/java/com/example/demo/dto/user/UserLoginRequestDto.java

```diff
@@ -0,0 +1,16 @@
+package com.example.demo.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotEmpty;
+import org.hibernate.validator.constraints.Length;
+
+public record UserLoginRequestDto(
+        @NotEmpty
```

Comment:
for String use `@NotBlank`

```suggestion
        @NotBlank
```